src/Compilers/CSharp/Portable/CodeGen/EmitExpression.cs

```diff
@@ -355,7 +355,7 @@ private void EmitLoweredConditionalAccessExpression(BoundLoweredConditionalAcces
         {
             var receiver = expression.Receiver;
 
-            if (receiver.IsDefaultValue())
+            if (receiver.IsDefaultValue() && receiver.Type.IsReferenceType)
```

Comment:
I didn't understand when this case can be reached. 
Per `RewriteConditionalAccess` below (in `LocalRewriter_ConditionalAccess.cs`), it seems that a conditional access to the default value for a reference type is lowered as simply a `BoundDefaultOperator`. Then how we get to the emit stage with a `BoundLoweredConditionalAccess` that matches those same conditions?